exercises/concept/cars-assemble/cars_assemble_test.go

```diff
@@ -7,46 +7,53 @@ import (
 
 const FloatEqualityThreshold = 1e-9
 
-func TestSuccessRate(t *testing.T) {
+func TestCalculateWorkingCarsPerHour(t *testing.T) {
 	tests := []struct {
-		name  string
-		speed int
-		want  float64
+		name           string
+		productionRate int
+		successRate    float64
+		want           float64
 	}{
 		{
-			name:  "calculate success rate for speed zero",
-			speed: 0,
-			want:  0.0,
+			name:           "calculate working cars per hour for production rate 0",
+			productionRate: 0,
+			successRate:    100,
+			want:           0.0,
 		},
 		{
-			name:  "calculate success rate for speed one",
-			speed: 1,
-			want:  1.0,
+			name:           "calculate working cars per hour for 100%% success rate",
+			productionRate: 221,
+			successRate:    100,
+			want:           221.0,
 		},
 		{
-			name:  "calculate success rate for speed four",
-			speed: 4,
-			want:  1.0,
+			name:           "calculate working cars per hour for 80%% success rate",
+			productionRate: 426,
+			successRate:    80,
+			want:           340.8,
 		},
 		{
-			name:  "calculate success rate for speed seven",
-			speed: 7,
-			want:  0.9,
+			name:           "calculate working cars per hour for 20.5%% success rate",
+			productionRate: 6824,
+			successRate:    20.5,
+			want:           1398.92,
 		},
 		{
-			name:  "calculate success rate for speed nine",
-			speed: 9,
-			want:  0.77,
+			name:           "calculate working cars per hour for 0%% success rate",
+			productionRate: 8000,
+			successRate:    0,
+			want:           0,
 		},
 	}
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got := SuccessRate(tt.speed)
+			got := CalculateWorkingCarsPerHour(tt.productionRate, tt.successRate)
 			if math.Abs(got-tt.want) > FloatEqualityThreshold {
```

Comment:
To ensure some consistency on the track, implement this float comparison like the [interest is interesting](https://github.com/exercism/go/blob/main/exercises/concept/interest-is-interesting/interest_is_interesting_test.go#L10) exercise.

Reply:
I've updated the test to use the same pattern.